docs/setup_visualstudio.md

```diff
@@ -350,75 +276,121 @@ Edit "Command Arguments" and click "OK".
 
 <img src="images/visualstudio146.png" width="768px" />
 
+<div class="primer-spec-callout warning" markdown="1">
+**Pitfall:** Set a breakpoint on the last line of `main` to keep the output window open.
 
-## Arguments and options
-
-<div class="primer-spec-callout info" markdown="1">
-Skip this subsection for EECS 280 project 1.  You'll need it for project 2 and beyond.
+With input redirection configured, Visual Studio will automatically close the terminal window, and you'll miss seeing the output.
```

Comment:
IMO it would be clearer to reorder the two sentences in this pitfall box. i.e. missing the output is the pitfall, the sentence that follows is the fix.